### ISSUE TITLE
Automatic Events disparity between Obj-C and Swift

### ISSUE THREAD

**Issue by jonahollman** [mixpanel/mixpanel-swift#222]
There are three events that are collected automatically by the Objective-C SDK, but are not collected by the Swift SDK.

The events are:
- Total App Sessions
- Total App Session Length
- First App Open Date

**Comment by zihejia** [comment]
Hi @jonahollman, thanks for raising this issue. We will add it in the next release.

**Comment by NaturalLam** [comment]
@zihejia what's the timeline for fixing that? We rely pretty heavily on those events.

**Comment by zihejia** [comment]
Yes, we've been working on it, the PR #254 still needs some work, will give you update shortly.

**Comment by NaturalLam** [comment]
@zihejia , any update for this?

**Comment by pknex** [comment]
Any update?
And, when this is fixed, will it also address #275 (App Crashed events not tracked in the swift library)?

**Comment by zihejia** [comment]
Sorry about the delay. v2.5.7 has released and it is now supporting those properties now:
```
Total App Sessions
Total App Session Length
First App Open Date
```

https://github.com/mixpanel/mixpanel-swift/releases/tag/v2.5.7
